DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DSACampaignsServlet.java

```diff
@@ -125,4 +124,26 @@ public static Entity createEntityFromDSACampaign(DSACampaign DSACampaignObject)
 
         return DSACampaignEntity;
     }
+
+    // Retrieves a unique DSA campaign id from datastore.
+    public static String getDSACampaignId() {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Query query = new Query("numDSACampaigns");
```

Comment:
Wouldn't it be better to assign a number to each campaign and query the campaigns for a unique ID?